### ISSUE TITLE
Allow feature to stay on

### ISSUE THREAD

**Issue by brentonhouse** [kritollm/API-Access#1]
Right now, you have to turn it on after you browse to page but before the ajax call is made on the page.  If you are too late, you will get errors.  It would be nice to be able to turn the feature on and leave it on even it you browse to a new page or refresh the page.

**Comment by kritollm** [comment]
Yes, I made it simple without options so it turns off to avoid break some sites. If you want it to auto enable on your site you can include the enableCrossOrigin.js. It's made to ease developing in localhost and to make web apps who can communicate with APIs who isn't CORS enabled. I can add an option menu later. I was in fact blocked from a site for suspect activity when using an earlier version and forgot to turn it off.

**Comment by brentonhouse** [comment]
ok.  I tried that with no luck.  I will look into other options...

**Comment by kritollm** [comment]
Ok. Very strange. If you include the script you will see the icon turns green on the first cross domain request.

**Comment by brentonhouse** [comment]
I will see if I can install the older version.  I didn't have near as many issues with that version.

**Comment by kritollm** [comment]
The difference between the new and older version is that it removes the Origin and the User-Agent header. The reason is that the extension was made to communicate with APIs in browser where you normally must create an app or communicate through a server. I will of course help, but I do not quite know what you're trying to achieve.

**Comment by brentonhouse** [comment]
I have tried putting the file on the server, putting it on the server and referencing it with a script tag, and a few other things.   I am just trying to run it on a localhost server right now.

**Comment by kritollm** [comment]
I have used it with angular both on localhost and on server. As a test you can copy past the source in a script tag in case there is som path problems.
